flax/experimental/nnx/nnx/state.py

```diff
@@ -229,25 +223,20 @@ def __sub__(self, other: 'State') -> 'State':
 def _state_flatten_with_keys(x: State):
   items = sorted(x._mapping.items(), key=lambda item: item[0])
   children = tuple((jtu.DictKey(key), value) for key, value in items)
-  return children, tuple(x._mapping.keys())
+  return children, tuple(key for key, _ in items)
 
 
 def _state_unflatten(
-  static: tp.Tuple[Path, ...] | None,
+  static: tp.Tuple[Path, ...],
```

Comment:
Use the built in `tuple` here and elsewhere.

Reply:
wdym?